packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/diagram/ViewDiagramDescriptionConverter.java

```diff
@@ -98,20 +99,19 @@ public class ViewDiagramDescriptionConverter implements IRepresentationDescripti
 
     private final Function<VariableManager, String> semanticTargetLabelProvider;
 
-    private final Function<DiagramElementDescription, String> idProvider = (diagramElementDescription) -> {
-        // DiagramElementDescription should have a proper id.
-        return UUID.nameUUIDFromBytes(EcoreUtil.getURI(diagramElementDescription).toString().getBytes()).toString();
-    };
+    private final Function<org.eclipse.sirius.components.view.DiagramElementDescription, String> idDiagramElementProvider;
```

Comment:
I'm not sure that you need to keep that and the diagram id provider above. Keep one only